bookdown/03-optimization-nested-resampling.Rmd

```diff
@@ -51,12 +51,18 @@ task = tsk("iris")
 learner = lrn("classif.rpart")
 resampling = rsmp("holdout")
 measure = msr("classif.ce")
-param_set = ps(cp = p_dbl(lower = 0.001, upper = 0.1))
+search_space = ps(cp = p_dbl(lower = 0.001, upper = 0.1))
 terminator = trm("evals", n_evals = 5)
 tuner = tnr("grid_search", resolution = 10)
 
-at = AutoTuner$new(learner, resampling, measure = measure,
-  param_set, terminator, tuner = tuner)
+at = AutoTuner$new(
+  learner = learner, 
+  resampling = resampling, 
+  measure = measure,
+  terminator = terminator, 
+  tuner = tuner, 
+  search_space = search_space
+)
```

Comment:
this is a bit verbose imho

Reply:
i think @be-marc mainly changed it because the order of the arguments changed. You can shorten it I guess.